src/probe/probe.c

```diff
@@ -163,6 +173,29 @@ static int probe_dma_deinit(struct probe_dma_ext *dma)
 	return 0;
 }
 
+static enum task_state probe_task(void *data)
```

Comment:
Please add some comment here describing this function.

Reply:
done